dialects/sqlite-3-33/src/test/fixtures_sqlite_3_33/update-set-from/1.s

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE test(
+  id TEXT
+);
+
+CREATE TABLE test2(
+  id2 TEXT
+);
+
+UPDATE test
+SET id = id2
+FROM (
+  SELECT
+    id2
+  FROM test2
+);
+
+UPDATE test
+SET id = newId
+FROM (
+  SELECT
+    id2 AS newId
+  FROM test2
+);
+
+UPDATE test
+SET id = newId
+FROM (
+  SELECT
+    id AS newId
+  FROM test
+);
+
+UPDATE test
+SET id = newId
+FROM (
+  SELECT
+  otherTest.id AS newId
+  FROM test2
+  INNER JOIN test AS otherTest
+  ON otherTest.id = test2.id2
+);
+
+UPDATE test
+SET id = newId
+FROM (
+  SELECT test.id AS newId
+  FROM test2
+  INNER JOIN test
+  ON test.id = test2.id2
+);
```

Comment:
This case fails with `1.s line 49:10 - Multiple columns found with name id` even though this works on sqlitebrowser. Wasn't sure how to fix that.